src/lib/abacus/index.ts

```diff
@@ -199,6 +200,10 @@ class AbacusStateManager {
       this.setTradeValue({ value: null, field: TradeInputField.leverage });
     }
 
+    if (uiRefresh && needsTargetLeverage) {
```

Comment:
Why not just clear them all the time? I worry about edge cases where target leverage is entered but then needsTargetLeverage becomes false before this clear function is called and we just leave it there forever.

Reply:
ah good point- will update!